src/modules/dashboard/DashboardEModeButton.tsx

```diff
@@ -195,16 +206,46 @@ export const DashboardEModeButton = ({
             </Trans>
           </Typography>
 
-          <Button
-            variant={isEModeDisabled ? 'gradient' : 'outlined'}
-            onClick={() => {
-              onClick();
-              handleClose();
-            }}
-            data-cy={`${isEModeDisabled ? 'emode-enable' : 'emode-disable'}`}
-          >
-            {isEModeDisabled ? <Trans>Enable E-Mode</Trans> : <Trans>Disable E-Mode</Trans>}
-          </Button>
+          {isEModeDisabled ? (
+            <Button
+              sx={{ width: '100%' }}
```

Comment:
I believe the same effect can be accomplished with the `fullWidth` MUI prop on these `Button` components

Reply:
Yep, would be cleaner, will change

Reply:
Updated